apps/fp-frontend-app/src/app/AppIndex.tsx

```diff
@@ -129,6 +129,26 @@ const createQueryClient = (errorHandler: (error: Error) => void) =>
     }),
   });
 
+const ZERO_RETRIES = false;
+
+const retryHandler = () => {
+  if (import.meta.env.MODE === 'test') {
+    return ZERO_RETRIES;
+  }
+
+  return (failureCount: number, error: Error) => {
+    if (error instanceof HTTPError) {
+      if (error.response.status === 401 || error.response.status === 403) {
+        return ZERO_RETRIES;
+      }
+      if (error.response.status === 500) {
+        return failureCount < 1;
+      }
+    }
+    return failureCount < 3;
+  };
```

Comment:
Kanskje jeg ikke tenker helt riktig men vil ikke `failureCount` begynne på 1 og derfor vil `failureCount < 1;`  alltid være false? så da kan det kombineres med håndteringen av 401 og 403

og for `failureCount < 3` vil det bare gi to retries? er det nok?

```suggestion
  const NO_RETRY_RESPONSECODES = [401, 403, 500];
  return (failureCount: number, error: Error) => {
    if (error instanceof HTTPError) {
      if (NO_RETRY_RESPONSECODES.includes(error.response.status)) {
        return ZERO_RETRIES;
      }
     }
    return failureCount <= 3;
  };
```

Reply:
Eg har testa. Den startar på 0